tgui/packages/tgui/interfaces/OrbitalCannonConsole.js

```diff
@@ -0,0 +1,150 @@
+              <b> {data.incfuel} Solid Fuel blocks.
+              </b><br />
+              Cluster Orbital Warhead:
+              <b> {data.clusterfuel} Solid Fuel blocks.
+              </b>
+            </Box>
+          </Collapsible>
+        </Section>
+        <Section title="Tray status">
+          {timeLeft > 0 && (
+            <ProgressBar
+              width="100%"
+              value={timeLeftPct}
+              ranges={{
+                good: [-Infinity, 0.33],
+                average: [0.33, 0.67],
+                bad: [0.67, Infinity],
+              }}>
+              <Box textAlign="center">
+                {Math.ceil(timeLeft / 10)} seconds
+                until the cannon can be chambered!
+              </Box>
+            </ProgressBar>
+          )}
+          {!data.loadedtray && (
+            <Button
+              fontSize="20px"
+              textAlign="center"
+              fluid={1}
+              disabled={!fullyLoaded}
+              icon="truck-loading"
+              color="good"
+              content="Load tray"
+              onClick={() => act('load_tray')}
+            />
+          ) || (
+            <Box>
+              {!data.chamberedtray && (
+                <Button
+                  fontSize="20px"
+                  textAlign="center"
+                  fluid={1}
+                  icon="sign-out-alt"
+                  color="good"
+                  content="Unload tray"
+                  onClick={() => act('unload_tray')}
+                />
+              ) || (
+                <NoticeBox
+                  fontSize="15px"
+                  textAlign="center"
+                  fluid={1}
+                  danger
+                >
+                  The tray is chambered, you cannot unchamber it.
+                </NoticeBox>
+              )}
+            </Box>
+          )}
+          {(!data.chamberedtray && !!data.loadedtray) && (
+            <Button.Confirm
+              fontSize="20px"
+              textAlign="center"
+              disabled={!!cantChamber || !fullyLoaded}
+              fluid={1}
+              icon="sign-in-alt"
+              color="good"
+              content="Chamber tray"
+              confirmContent="You cannot unchamber the tray. Confirm?"
+              onClick={() => act('chamber_tray')}
+            />
+          )}
+          {!!data.disabled && (
+            <Dimmer fontSize="32px">
+              <Icon name="exclamation-triangle" />
+              {' Cannon disabled!'}
+            </Dimmer>
+          )}
+          {!data.linkedcannon && (
+            <Dimmer fontSize="32px">
+              <Icon name="exclamation-triangle" />
+              {' No cannon linked to console!'}
+            </Dimmer>
+          )}
+          {!data.linkedtray && (
+            <Dimmer fontSize="32px">
+              <Icon name="exclamation-triangle" />
+              {' No tray linked to console!'}
+            </Dimmer>
+          )}
```

Comment:
just change the text, no need to have the same block 3 times

Reply:
done